server/src/services/sqlIngest.py

```diff
@@ -84,19 +84,21 @@ def cleanData(self):
 
     def ingestData(self, ingestMethod='replace'):
         '''Set up connection to database'''
-        print('Inserting data into Postgres instance...')
+        print('Inserting data into MySQL instance...')
```

Comment:
Can you replace this with the dialect portion of the connection string value from config?
`Dbstring.split(':')[0]` or something like that

Reply:
on it

Reply:
My recommendation would be to split and make a new self.dialect variable or something like that under __init__ - then we won't have to add a variable to each method call.